Covinator/Covinator/Pages/modernaVaccination.cshtml

```diff
@@ -1,34 +1,42 @@
 ﻿@page
 @model Covinator.Pages.modernaVaccinationModel
 @{
-    var modernaVaccineDistributionAllocationss = (QuickType.ModernaVaccineDistributionAllocations[])ViewData["ModernaVaccineDistributionAllocations"];
+    ViewData["Title"] = "Moderna Vaccination";
+    var modernaAllocations = (QuickType.ModernaVaccineDistributionAllocations[])ViewData["ModernaVaccineDistributionAllocations"];
 }
 
 <div class="text-center">
     <h1 class="display-4">Moderna Vaccine Distribution Allocations</h1>
     <hr />
-    <table border="1" weidth="90%">
-        <tr>
-            <th>Jurisdiction</th>
-            <th>Week of allocations</th>
-            <th>1st dose allocations</th>
-            <th>2nd dose allocations</th>
-            @{ foreach (QuickType.ModernaVaccineDistributionAllocations modernaVaccineDistributionAllocations in modernaVaccineDistributionAllocationss)
-                {
-                <tr>
-                    <td>@modernaVaccineDistributionAllocations.Jurisdiction </td>
-                    <td>@modernaVaccineDistributionAllocations.WeekOfAllocations of allocations </td>
-                    <td>@modernaVaccineDistributionAllocations.The1StDoseAllocations </td>
-                    <td>@modernaVaccineDistributionAllocations.The2NdDoseAllocations </td>
 
-                </tr>
- } }
-
-
-
-        </table>
-
-
-
-    </div>
+    @{
+        if (modernaAllocations.Length > 0)
+        {
+            foreach (QuickType.ModernaVaccineDistributionAllocations modernaAllocation in modernaAllocations)
+            {
+                <table border="1" weidth="90%">
+                    <tr>
+                        <th>Jurisdiction</th>
+                        <th>Week of allocations</th>
+                        <th>1st dose allocations</th>
+                        <th>2nd dose allocations</th>
+
+                    <tr>
+                        <td>@modernaAllocation.Jurisdiction </td>
+                        <td>@modernaAllocation.WeekOfAllocations</td>
+                        <td>@modernaAllocation.The1StDoseAllocations </td>
+                        <td>@modernaAllocation.The2NdDoseAllocations </td>
+
+                    </tr>
+
+
+                </table>
+            }
+        }
+        else
+        {
+            <h1 class="display-4">No Data to Display</h1>
```

Comment:
Display message to show no data instead of showing empty table with header.

Source:-
https://docs.microsoft.com/en-us/aspnet/core/razor-pages/?view=aspnetcore-5.0&tabs=visual-studio